ext/sodium/libsodium_arginfo.h

```diff
@@ -348,6 +348,11 @@ ZEND_END_ARG_INFO()
 
 #define arginfo_sodium_crypto_stream_xor arginfo_sodium_crypto_secretbox
 
+#ifdef crypto_stream_xchacha20_KEYBYTES
+#define arginfo_sodium_crypto_stream_xchacha20_xor arginfo_sodium_crypto_stream_xor
+#define arginfo_sodium_crypto_stream_xchacha20 arginfo_sodium_crypto_stream
+#endif
```

Comment:
This is a generated file, run build/gen_stub.php ext/sodium to regenerate.

Reply:
This function isn't guaranteed to exist on older versions ([1.0.11 and earlier](https://libsodium.gitbook.io/doc/advanced/stream_ciphers/xchacha20)) of libsodium. Does `build/gen_stub.php` need to take this into consideration?

Reply:
Yes, you need to add the corresponding `#ifdef` in the stub file as well. They will be transferred to the generated output.